packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -1079,3 +1078,180 @@ input:-webkit-autofill {
   flex-direction: row;
   justify-content: center;
 }
+
+.EventTypeTradingPairsRow__container {
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+}
+
+.EventTypeTradingPairsRow__container > * {
+  margin-bottom: 0 !important;
```

Comment:
i notice we do this a lot. i wonder if it may have been better to address the :not(:last-child) instead haha

Reply:
haha yeah probably, we should clean it up